### PR TITLE
feat: add doc for ldap auth

### PR DESCRIPTION
**Pull request by nexdrew**
A new page for LDAP Auth that gives a brief description of every field and includes a table with example values at the end.

### CODE REVIEW

File: up-and-running/auth/ldap.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+npm Enterprise supports most LDAP corporate directories, including Active Directory and OpenLDAP.
+
+It's important to note that, when using LDAP integration, your npm Enterprise instance(s) should typically be running on the same subnet as your LDAP servers.
+
+Note that you can also use LDAP integration for logging into the admin console (port `:8800`) as well. If you'd like to set that up, visit the `/create-password` page once you're logged in, choose the `LDAP`, and enter the settings just as you would below.
+
+## Configuration Fields
+
+### Server Settings
```
Comment (bcoe) [inline up-and-running/auth/ldap.md]: It's a little weird seeing three headings in a row, does this look nice when rendered in GitBook? I like the structure.

File: up-and-running/auth/ldap.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+#### Encryption Type
+
+_Required._ The type of encryption your LDAP provider supports.
+
+Options are `Plain`, `StartTLS`, and `LDAPS`.
+
+#### Search user
+
+_Required._ The user, in distinguished name (DN) format, that npm Enterprise should use to log into LDAP. When users attempt to authenticate against the private npm registry or website, npm Enterprise will use this user to query the corporate directory service. This should typically include at least one common name (CN) representing a read-only service or admin user.
```
Comment (bcoe) [inline up-and-running/auth/ldap.md:35]: Great description, I feel like I understand more about this field now ... I feel like the help I've given people on this topic has always been slightly hand-wavy.

File: up-and-running/auth/ldap.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+_Required._ The tree node, in distinguished name (DN) format, relative to the "Base DN" above, that all npm Enterprise users should belong to. This is typically a single common name (CN) or organizational unit (OU).
+
+#### Restricted User Group
+
+_Optional._ A group name that users must be a part of, used as an additional criterion in the LDAP query when looking up users. This is typically **not** a distinguished name (DN).
+
+#### Username field
+
+_Required._ The attribute of a user entry that represents the username to be used when authenticating. Active Directory uses `sAMAccountName` as its default.
```
Comment (bcoe) [inline up-and-running/auth/ldap.md:57]: does LDAP try to have opaque variable names?

File: up-and-running/auth/ldap.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+#### Test password
+
+The password value to test LDAP configuration with.
+
+## Example Configuration
+
+| Config Field          | Example Value                                       |
+| --------------------- | --------------------------------------------------- |
+| LDAP Type             | `Active Directory`                                  |
```
Comment (bcoe) [inline up-and-running/auth/ldap.md:75]: I like that you included an example 👍

**Review by bcoe — APPROVED** [review]
This is great, I'm glad we can finally direct people to something.

File: up-and-running/auth/ldap.md (hunk 1 above)
Comment (nexdrew) [reply up-and-running/auth/ldap.md]: Yeah, I was trying to organize the fields like they are in the admin console while still allowing for individual fields to be linked. Top of the page looks like this, let me know what you think @bcoe:
screen shot 2016-10-03 at 6 22 17 pm

File: up-and-running/auth/ldap.md (hunk 3 above)
Comment (nexdrew) [reply up-and-running/auth/ldap.md:57]: Of course! It's the sign of a good protocol, didn't you know?

**Pull request merged**